2.0/backup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# This script creates a full backup of the mysql database
+# and rolls backup files over
+#
+set -e
+
+# Configuration
+BKUP_DIR=${BKUP_DIR:-"/snapshots"}
+BKUP_RETENTION=${BKUP_RETENTION:-20}
+
+# Ensure backup directory exists
+mkdir -p $BKUP_DIR
+
+# Perform backup
+cd /tmp
+ts=$(date -u +"%Y-%m-%dT%H-%M-%SZ")
+tar -zcf $ts.tar.gz /export/
```

Comment:
See https://github.com/minio/minio/issues/4398:
> 1. In general it is NOT safe to access the backend file system when minio could be doing I/O
> 3. What we'll likely end up doing for backup is using `mc mirror` to put a copy of the object store into the local filesystem and then backup/snapshot that filesystem using our tools of choice

It should be ok in our case as we only do I/O when publishing the new style or fetching the assets (at restart)
But that's something to keep in mind if decide to use for other purpose.